.github/workflows/stale.yml

```diff
@@ -44,3 +44,4 @@ jobs:
         days-before-pr-close: '20'
         days-before-issue-stale: '60'
         days-before-issue-close: '67'
```

Comment:
I'm pretty sure this value is wrong though:

> The number of days to wait to close an issue or a pull request after it being marked stale

We should set this as 7 right? Not 67?

Reply:
Yes. I thought it was the total days, not days since the stale label was added.